pkg/tsdb/opentsdb/opentsdb.go

```diff
@@ -408,6 +429,10 @@ func (s *Service) buildMetric(query backend.DataQuery) map[string]any {
 		metric["filters"] = model.Filters
 	}
 
+	if model.ExplicitTags {
```

Comment:
since the assigned value is also a `boolean` can't this just be `metric["explicitTags"] = model.ExplicitTags` not sure if there is a downside / problem with assigning that to false though.